packages/cardano-services/blockfrost-keys/README.md

```diff
@@ -0,0 +1,7 @@
+### Blockfrost API keys
+
+Store here the files containing the Blockfrost API key for each network.
```

Comment:
This breaks the 'self-containment' of the application. Previously all files required to run it were under `dist/`. A cleaner approach would be using env variables, but if we don't want to do that then we should at least copy the keys to 'dist/cjs/' during `yarn build`

Reply:
The existing strategy for sensitive information is to offer both secrets support and ENV

Reply:
Thanks for an explanation, no change required